modules/openxr/doc_classes/OpenXRDpadBindingModifier.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="OpenXRDpadBindingModifier" inherits="OpenXRBindingModifier" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../doc/class.xsd">
+	<brief_description>
+		The DPad binding modifier converts an axis input to a dpad output.
+	</brief_description>
+	<description>
+		The DPad binding modifier converts an axis input to a dpad output, emulating a DPad. New input paths for each dpad direction will be added to the interaction profile. When bound to actions the DPad emulation will be activated. You should [b]not[/b] combine dpad inputs with normal inputs in the same action set for the same control, this will result in an error being returned when suggested bindings are submitted to OpenXR.
+		See [url=https://registry.khronos.org/OpenXR/specs/1.1/html/xrspec.html#XR_EXT_dpad_binding]XR_EXT_dpad_binding[/url] for in-depth details.
+		[b]Note:[/b] If the DPad binding modifier extension is enabled, all dpad binding paths will be available in the action map. Adding the modifier to an interaction profile allows you to further customize the behavior.
+	</description>
+	<tutorials>
+	</tutorials>
+	<members>
+		<member name="action_set" type="OpenXRActionSet" setter="set_action_set" getter="get_action_set">
+			Action set for which this dpad binding modifier is active.
+		</member>
+		<member name="center_region" type="float" setter="set_center_region" getter="get_center_region" default="0.1">
+			Center region in which our center position of our dpad return [code]true[/code].
+		</member>
+		<member name="input_path" type="String" setter="set_input_path" getter="get_input_path" default="&quot;&quot;">
+			Input path for this dpad binding modifier.
+		</member>
+		<member name="is_sticky" type="bool" setter="set_is_sticky" getter="get_is_sticky" default="false">
+			If [code]false[/code], when the joystick enters a new dpad zone this becomes true.
+			If [code]true[/code], when the joystick remains in active dpad zone, this remains true even if we overlap with another zone.
```

Comment:
Also hard to understand what this field is for at first read.

Reply:
I had real difficulty describing this in text, you really need to look at the diagrams in the API spec and even there the description is overly technical.

Reply:
Are those diagrams publicly available? Can you link to them in this description?

Reply:
If you scroll to the top of the help page you'll notice that on both modifiers, I link to the specification for the modifiers.

For the DPad modifier its this one: https://registry.khronos.org/OpenXR/specs/1.1/html/xrspec.html#XR_EXT_dpad_binding

Totally open to suggestions on how to work that more user friendly and embed that into our documentation.